corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -701,7 +701,7 @@ hqDefine('cloudcare/js/backbone/apps.js', function () {
                 $.ajax({
                     type: 'POST',
                     url: submitUrl,
-                    data: xml,
+                    data: xml.output,
```

Comment:
:+1:

On Wed, May 18, 2016 at 5:09 PM, wpride <notifications@github.com> wrote:

> In corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js
> <https://github.com/dimagi/commcare-hq/pull/11644#discussion_r63782690>:
>
> > @@ -701,7 +701,7 @@ hqDefine('cloudcare/js/backbone/apps.js', function () {
> >                  $.ajax({
> >                      type: 'POST',
> >                      url: submitUrl,
> > -                    data: xml,
> > +                    data: xml.output,
>
> see change here https://github.com/dimagi/touchforms/pull/221/files
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/11644/files/2dbcf7c059d3151f621961fec5b34773f8c85e2f#r63782690>
>